### ISSUE TITLE
error on Catalina if .config folder is stored in iCloud

### ISSUE THREAD

**Issue by nealeyoung** [pqrs-org/Karabiner-Elements#2032]
My config file (in fact entire ~/.config directory) is stored in iCloud via symbolic link into ~/Documents.  Karabiner is not working for me under Catalina, and according to the log is failing to read the config file:

`[2019-09-08 14:55:48.973] [error] [grabber] failed to open /Users/XXX/.config/karabiner/karabiner.json`

Apparently moving the .config folder to local storage fixes the problem.

This issue was also posted about in a related thread about Karabiner on Catalina not working, by @cumanzor in https://github.com/tekezo/Karabiner-Elements/issues/1867#issuecomment-529242909_

But that thread is mainly about a different issue.  Presumably this issue can be fixed by somehow triggering Mac OS to ask to grant permissions for Karabiner to read the config file in this case.

**Comment by nealeyoung** [comment]
I was able to work around the issue by enabling "Full Disk Access" for karabiner_grabber in System Prefs -> Security & Privacy -> Privacy -> Full Disk Access, see attached screen shot.  Karabiner-grabber was already in the list, but the box was not checked.  I checked the box.

Presumably it would be preferable to enable access to just the config file, via the "Files and Folders" access options, but that doesn't seem to be an option.

![Untitled](https://user-images.githubusercontent.com/965060/67112035-4ec56280-f1a4-11e9-877b-72b587d19409.png)
